draft-ietf-rats-corim.md

```diff
@@ -1186,6 +1194,39 @@ Evidence.
 {::include cddl/domain-membership-triple-record.cddl}
 ~~~
 
+
+
+#### Multi-Environment Conditional (MEC) Endorsements Triple {#sec-comid-triple-mec-endorsements}
+
+The semantics of the Multi-Environment Conditional (MEC) Endorsements Triple is as follows:
+
+> "IF accepted state matches the `cond` value, THEN `env` is associated with the endorsed value(s) `ends`."
+
+~~~ cddl
+{::include cddl/mec-endorsement-triple-record.cddl}
+~~~
+
+A `multi-env-conditional-endorsement-triple-record` has the following parameters:
+
+* `conds`: all target environments, along with a specific state, that need to match in order for the endorsement(s) to apply
+* `actions`: TODO 
+* `env`: the environment to which the endorsed value (conditionally) applies
+* `ends`: the endorsed value(s) associated with `env`
```

Comment:
these two should be dropped

Reply:
addressed in https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/174/commits/f553dd364fc1dea0e1663f183555c45e92c96103